cli/README-cn.md

```diff
@@ -0,0 +1,718 @@
+3. 用[Cordova插件](http://github.com/Microsoft/cordova-plugin-code-push) 或 [React Native插件](http://github.com/Microsoft/react-native-code-push)配置好CodePush并指向你希望的部署环境
+4. [发布](#发布更新)更新
+5. 活的长而成功！[详细资料](https://en.wikipedia.org/wiki/Vulcan_salute)
+
+## 创建账号
+
+在你发布应用更新之前，你需要创建一个CodePush帐号。一旦你安装了Cli你就可以简单的使用如下命令来注册：
+
+```
+code-push register
+```
+
+这将会启动浏览器，要求验证你的Github或微软帐号。一旦验证成功，它将创建一个CodePush帐号跟你的Github或MSA相连，并生成一个访问密钥(Access Key)，你可以拷贝/粘贴到CLI以便登录。
+
+*注意：注册成功后，你就已经自动登录了。所以除非你明确登出了，否则你不需要在此机器上再次登录。*
+
+如果你已有一个帐号，那你还可以把你的帐号跟另一个身份认证提供商关联起来，通过运行：
+
+（我个人理解：CodePush提供商有Github和Mircosoft，它允许你可以把两个帐号关联起来。）
+
+```
+code-push link
+```
+
+注意：为了实现这个(关联)的目的，你在另一个身份认证供应商那边用的Email地址必须与你现存的帐号一致。
+
+## 身份认证
+
+在CodePush CLI里大多数命令需要身份认证，所以在你开始管理你的帐号之前，你需要使用GitHub或者微软帐号注册和登录。你可以通过执行如下命令做到这些：
+
+```
+code-push login
+```
+
+这将会启动浏览器，要求验证你的Github或微软帐号。这将生成一个访问密钥(Access Key)，然后你可以拷贝/粘贴到CLI（它会提示你这样做）。这时你就认证成功了，并且可以关掉你的浏览器了。
+
+
+如果在任何时候你想确认你是否已经登录了，你可以运行如下命令来显示与你当前认证会话相关的e-mail帐号，而且这个身份提供者是连接到（如：GitHub）的。
+
+```shell
+code-push whoami
+```
+
+当你从CLI登录后，你的访问密钥(Access Key)就一直保存在你本地磁盘上，所以你不必每次使用帐号是都需要登录。为了终止会话或删除AccessKey，你可以简单的运行如下命令：
+
+```
+code-push logout
+```
+
+如果你在一台机器上忘记注销（比如：你朋友的电脑上），你可以使用如下命令列出和删除任何“激活中”的Access Keys。Access Keys列表将显示创建Key的机器名和发生登录的时间。这让你可以简单的认出那些你不想要保存的Keys。
+
+```
+code-push access-key ls
+code-push access-key rm <accessKey>
+```
+
+如果你需要额外的Keys，被用来验证CodePush服务而不需要给你的GitHub和/或访问微软凭证，您可以运行下面的命令来创建一个持久的Access Key(连同一个描述):
+
+```
+code-push access-key add "VSTS Integration"
+```
+在创建新的密钥之后，您可以在`login`命令后使用`--accessKey`标志并指定其值，它允许您执行“无头”身份验证，而不是启动一个浏览器。
+
+```
+code-push login --accessKey <accessKey>
+```
+当使用这种方式登录时，密钥(Access Key)在注销时不会自动失效，它可以一直被使用，除非它从CodePush服务端明确被移除掉。然而，仍然建议一旦你完成了会话就注销掉，以便从本地磁盘移除掉你的授权证书。
+
+## 应用管理
+
+在你发布更新前，你需要用如下命令在CodePush服务上注册一个App：
+
+```
+code-push app add <appName>
+```
+
+如果你的App既有IOS又有Android，我们推荐你创建单独的App。一个平台一个。这样你可以单独的管理和发布更新，从长远来看这会让事情更简单。大部分人的命名约定会在App名加后缀`-IOS`和`-Android`。例如：
+
+```
+code-push app add MyApp-Android
+code-push app add MyApp-iOS
+```
+
+所有新的Apps自动会出现有个部署环境（`Staging`和`Production`），为了你可以开始发布更新到不同的渠道而不需要做任何其它的事（参考下面的部署指南）。你创建一个App之后，CLI将显示`Staging`和`Production`环境的开发密钥，你就可以使用不同的SDKs(详细请看[Cordova](http://github.com/Microsoft/cordova-plugin-code-push) 和 [React Native](http://github.com/Microsoft/react-native-code-push))来配置你的手机端App了。
+
+如果你不喜欢你之前取的名字，你还可以随时重命名它，使用如下命令：
+
+```
+code-push app rename <appName> <newAppName>
+```
+
+应用的名字从管理方面看只是为了能辨识，因此，必要时可以随时重命名它。它其实不会影响正在运行的应用程序，因为更新的查询都是通过部署密钥的。
+
+如果你不想要一个App，你可以从服务端上移除它，命令如下：
+
+```
+code-push app rm <appName>
+```
+
+做这个移除请务必小心，因为任何配置了它的App都将停止收到更新了。
+
+最后，如果你想列出你在CodePush服务上注册的所有Apps，你可以运行如下命令：
+
+```
+code-push app ls
+```
+
+## 应用合作
+
+如果你讲和其它开发者在一起合作同一个CodePush应用，你可以把他们添加为合作者，使用如下命令：
+
+```shell
+code-push collaborator add <appName> <collaboratorEmail>
+```
+
+*注意： 这个期待开发者已经用e-mail[注册](#创建账号)了CodePush，所以在打算分享应用之前确保他们已经准备好了一切。*
+
+一旦添加了，所有的合作者将立即拥有了最新分享App的如下权限：
+1. 查看App，它的合作者，[部署管理](#部署管理)和[查看发布历史](#查看发布历史)。
```

Comment:
You have to add an empty line above this line, otherwise the GitHub markdown will not render properly: https://github.com/quentinyang/code-push/blob/9691d18857374ab2380735a1e1ae78b63521be94/cli/README-cn.md#应用合作